### DISCUSSION TITLE
Fatture - creazione da Python

### DISCUSSION THREAD

**Discussion by iarin-fabbri** [fattureincloud/api#255] [Q&A] [answered]
Ciao a tutti,
sono nuovo di Fatture in cloud e devo sviluppare uno script per un mio cliente.
La mia domanda è semplice: se creo una fattura, questa viene inviata automaticamente all'SDI o è possibile creare una fattura in "bozza" e poi effettuare l'invio direttamente dal portale?

La mia esigenza è quella di creare delle fatture, ma queste non devono partire in automatico, perchè prima vano controllate sul portale (ed eventualmente modificate aggiungendo a mano articoli e/o servizi).
A fronte della mia esigenza, quale strada è meglio percorrere?

**Comment by valmoz** [comment, marked as answer]
Buongiorno,
la chiamata per l'invio all'SDI è la [Send the e-invoice](https://developers.fattureincloud.it/api-reference#post-/c/-company_id-/issued_documents/-document_id-/e_invoice/send). Se questa chiamata non viene effettuata la fattura non viene inviata all'SDI.

Le consiglio anche la [guida alla creazione delle fatture](https://developers.fattureincloud.it/docs/guides/invoice-creation) e la [guida alla gestione delle e-invoices](https://developers.fattureincloud.it/docs/guides/e-invoice-management).

A disposizione
Mauro

**Reply by iarin-fabbri** [reply]
Grazie mille.
Un'ulteriore domanda: una volta creata la fattura via API, questa può essere cancellata da portale?
Mi scuso se la domanda risulta banale, ma ho lavorato con ERP che una volta fatta la fattura via API, questa non era più modificabile e in questo caso devo organizzare lo sviluppo introducendo un ambiente di test.

**Reply by valmoz** [reply]
Buongiorno @iarin-fabbri 
può cancellare tranquillamente i documenti utilizzando la chiamata [Delete Issued Document](https://developers.fattureincloud.it/api-reference#delete-/c/-company_id-/issued_documents/-document_id-), a patto che non sia stata già inviata all'SDI.

Grazie
Mauro

**Reply by iarin-fabbri** [reply]
Grazie mille, ora ho tutte le info.
